ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -655,6 +682,16 @@ - (Class)questionResultClass {
     return [ORKChoiceQuestionResult class];
 }
 
+- (NSString*)stringForAnswer:(id)answer {
+    NSMutableArray<ORKTextChoice *> *textChoices = [[NSMutableArray alloc] init];
+    for (ORKImageChoice *imageChoice in _imageChoices) {
+        if (imageChoice.text) {
+            [textChoices addObject:[[ORKTextChoice alloc] initWithText:imageChoice.text detailText:nil value:imageChoice.value exclusive:false]];
+        }
+    }
+    return textChoices.count > 0 ? [self stringForAnswer:answer fromTextChoices:textChoices] : nil;
```

Comment:
I have an idea: Let both `ORKTextChoice` and `ORKImageChoice` adapt to a new protocol in the internal header.

```
@protocol ORKChoice

- (id<NSCopying, NSCoding, NSObject>) value;
- (NSString *)text;
@end
```

Then you can process both them  `[self stringForAnswer:answer fromTextChoices:textChoices] `

You might also need to update ORKChoiceAnswerFormatHelper a bit too.